### PR TITLE
Refactor setup to use es6 modules

### PR DESCRIPTION
**Pull request by nmaier95**

### CODE REVIEW

File: src/database/databaseFactory.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import mongodb from './adapter/mongo.js';
+
+export default class DatabaseFactory {
```
Comment (samsour) [inline src/database/databaseFactory.js]: This class does not really match a factory pattern. This creates Connections, not Databases.
Accordingly, think `DatabaseConnectionFactory` would be more fitting in this case.

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "extends": [
+        "google"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": ["error", 4],
```
Comment (samsour) [inline .eslintrc.json:15]: Does 4 spaces automatically match and transform into a `tab`?

**Review by samsour — COMMENTED** [review]
I have a few questions regarding eslint and the naming of the Database Connection Class.

File: .eslintrc.json (hunk 2 above)
Comment (nmaier95) [reply .eslintrc.json:15]: Four spaces "might" equal one tab. But not neccessarily. It depends on e.g. user IDE settings. But one space always equals one space. Therefore four spaces are more specific than one tab.

**Pull request merged**